conans/client/manager.py

```diff
@@ -327,21 +327,27 @@ def _load_deps_info(self, current_path, conanfile, output):
             except:
                 output.error("Parse error in '%s' file in %s" % (BUILD_INFO, current_path))
         else:
-            output.warn("%s file not found in %s\nIt is recommended for source, build and package "
-                        "commands\nYou can generate it using 'conan install -g env -g txt'"
-                        % (BUILD_INFO, current_path))
-
-        env_file = os.path.join(current_path, CONANENV)
-        if os.path.exists(env_file):
-            try:
-                deps_env_info = DepsEnvInfo.loads(load(env_file))
-                conanfile.deps_env_info = deps_env_info
-            except:
-                output.error("Parse error in '%s' file in %s" % (CONANENV, current_path))
-        else:
-            output.warn("%s file not found in %s\nIt is recommended for source, build and package "
-                        "commands\nYou can generate it using 'conan install -g env -g txt'"
-                        % (CONANENV, current_path))
+            if not for_imports:
+                output.warn("%s file not found in %s\nIt is recommended for source, build and package "
+                            "commands\nYou can generate it using 'conan install -g env -g txt'"
+                            % (BUILD_INFO, current_path))
+            else:
+                raise ConanException("%s file not found in %s\nIt is required for imports "
```

Comment:
OK. Next time feel free to ask me to make necessary changes in my submissions :)